docs/central-entities.rst

```diff
@@ -300,39 +307,15 @@ Updating Entities from Forms
 
 You can use Forms to update Entity information. These Forms can be authored to, for example, update previous observations to new values or change the status of an Entity. Just like Entity creation through Forms, you can specify which properties on which Entity instances will be updated when the Form's Submissions are uploaded to Central. The data in Submissions uploaded by Entity-updating Forms are applied to the Entity data saved on the Central server. These updated Entity values are then distributed to data collection clients once they synchronize with Central.
 
-.. _central-entities-update-conflicts:
-
-Parallel Updates in Updating Forms
```

Comment:
I'm a fan of moving this down and not having it be so prominent in the updating entities section!